lib/cylc/suite_events.py

```diff
@@ -137,20 +137,28 @@ def _run_event_custom_handlers(self, config, ctx):
         for i, handler in enumerate(handlers):
             cmd_key = ('%s-%02d' % (self.SUITE_EVENT_HANDLER, i), ctx.event)
             # Handler command may be a string for substitution
-            cmd = handler % {
-                'event': quote(ctx.event),
-                'suite': quote(ctx.suite),
-                'message': quote(ctx.reason),
-                'suite_url': quote(config.cfg['URL']),
-            }
+            abort_on_error = self.get_events_conf(
+                config, 'abort if %s handler fails' % ctx.event)
+            try:
+                cmd = handler % {
+                    'event': quote(ctx.event),
+                    'suite': quote(ctx.suite),
+                    'message': quote(ctx.reason),
+                    'suite_url': quote(config.cfg['URL']),
+                }
+            except KeyError as exc:
+                message = "%s bad template: %s" % (cmd_key, exc)
+                LOG.error(message)
+                ERR.error(message)
```

Comment:
If you log error to the `LOG` then it is copied to `ERR` by default so you will see duplicate messages here.